src/ca65/expr.c

```diff
@@ -417,6 +418,34 @@ static ExprNode* FuncDefined (void)
 
 
 
+static ExprNode* FuncDefinedInstr (void)
+/* Handle the .DEFINEDINSTR builtin function */
+{
+    int Instr = 0;
```

Comment:
Is

.definedopstr()

a possible choice? For "operation code"

MC

Am 19.05.2015 um 11:49 schrieb greg-king5:
>
> In src/ca65/expr.c 
> <https://github.com/cc65/cc65/pull/160#discussion_r30584208>:
>
> > @@ -417,6 +418,34 @@ static ExprNode* FuncDefined (void)
> >
> >
> >
> > +static ExprNode* FuncDefinedInstr (void)
> > +/* Handle the .DEFINEDINSTR builtin function */
> > +{
> > +    int Instr = 0;
>
> |FindInstruction()| returns |-1| if its argument isn't an instruction.
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/cc65/cc65/pull/160/files#r30584208>.
>